packages/ra-core/src/controller/ListController.spec.tsx

```diff
@@ -65,7 +65,7 @@ describe('ListController', () => {
 
             const { getByLabelText } = render(
                 <TestContext
-                    store={{
+                    initialState={{
```

Comment:
Why don't you use your shiny new `renderWithRedux` ? Same for all tests using TestContext

Reply:
I was afraid of the rebase ....

Reply:
What rebase ?